Tests/SwerverTests/TestData.swift

```diff
@@ -5,23 +5,23 @@ class TestData {
     static func validUrlOne() -> URL {
         return URL(string: "/")!
     }
-
+    
```

Comment:
Make sure to remove trailing whitespace. Most editors allow you to have that done automatically.

Reply:
will fix that

Reply:
fixed